### DISCUSSION TITLE
[Feature] Filter in and out albums from the photos page

### DISCUSSION THREAD

**Discussion by Atakku** [immich-app/immich#3153] [Feature Request]
### The feature

Would be great to be able to filter out images that do not belong to album X, but belong to album Y, and etc. 
Personally would help a lot when organizing unsorted library into albums, as this will provide a way to see images which are not in any of the albums.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1621
